src/workerd/api/crypto-impl-asymmetric.c++

```diff
@@ -1010,18 +1004,22 @@ kj::OneOf<jsg::Ref<CryptoKey>, CryptoKeyPair> CryptoKey::Impl::generateRsa(
   auto usages = CryptoKeyUsageSet::validate(normalizedName, CryptoKeyUsageSet::Context::generate,
                                             keyUsages, validUsages);
 
-  validateRsaParams(modulusLength, publicExponent.asPtr());
+  validateRsaParams(js, modulusLength, publicExponent.asPtr());
+  // boringssl silently uses (modulusLength & ~127) for the key size, i.e. it rounds down to the
+  // closest multiple of 128 bits. This can easily cause confusion when non-standard key sizes are
+  // requested.
+  // The `modulusLength` field of the resulting CryptoKey will be incorrect when the compat flag
+  // is disabled and the key size is rounded down, but since it is not currently used this is
+  // acceptable.
+  if (FeatureFlags::get(js).getStrictCrypto()) {
+    JSG_REQUIRE(!(modulusLength & 127), DOMOperationError,
```

Comment:
Here too... can the Feature flag check be included in the `JSG_REQUIRE` condition?